pragma/views/endpoints.py

```diff
@@ -24,12 +24,6 @@ def get(self, oaid=None):
             return OpenAnnotation.get(oaid).dict()
         return {"annotations": [oa.dict() for oa in OpenAnnotation.query.all()]}
 
-    @rest
```

Comment:
@JesseWeinstein why is this section being removed? I believe this is what the iiif.archivelab.org Mirador viewer is using to create non-wayback annotations for our books.